scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml

```diff
@@ -8,11 +8,11 @@ shellcheck_target_version: "0.4.6"
 cmake_prefix: "/usr/local"
 
 #TODO: Flip to Clang-8 after https://github.com/openenclave/openenclave/pull/3574 is merged
-clang_binary_name: "clang-7"
```

Comment:
On line 6 of this file, we need to change the clang_target_version to 8.0.0

Reply:
Done as suggested. Should the version be clang 8.0.1 or clang 8.0.0?
Not sure which version of clang CI was updated to
https://releases.llvm.org/download.html

Reply:
Thanks.

When I ran the validation locally, I saw 8.0.0